### ISSUE TITLE
Provide test the new average read transaction time

### ISSUE THREAD

**Issue by codeDavidLopez** [amosproj/amos2022ss06-idunion-blockchain-dashboard#55]
## User story
1. As a developer
2. I want / need a test for the new average read transaction time feature
3. So that regression testing is available

## Acceptance criteria
* the test checks the feature
* test added to test suite for regression testing

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## Feature DoD 
* Feature has been fully implemented
* All acceptance criteria were met
* All tests are passing
* No known errors
* Documentation updated
* Bill of materials updated

## Sprint Release DoD
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Peer code review passed
* Developers agreed to release
* Documentation updated
* Bill of materials updated
